releases/README.md

```diff
@@ -42,21 +42,27 @@
 3. Push tag and Github release
 
    1. Push the tag to `git@github.com:containerd/containerd.git`.
-      NOTE: this will kick off CI building of the release binaries.
+      NOTE: this will kick off CI to prepare the Github release and binaries.
 
-   2. Create the Github release using the `Tag version` which was just pushed. Use the first
-      line outputted from the release tool as the `Release title` and the remainder of the
-      output for the description. No alteration of the release output should be needed.
-      Ensure `pre-release` is checked if an `-rc`.
```

Comment:
I notice this was removed (ensure pre-release is checked); was there a reason to remove it?

(Or did GitHub fix this to do it automatically based on version now?)

Reply:
GitHub CI does it for us automatically. We do not manually create the releases anymore.